ControlsKit/CTKSwitch.h

```diff
@@ -43,7 +43,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  Background color when the switch is turned off.
  */
-@property (nonatomic, strong, nullable) IBInspectable UIColor* offTintColor;
+@property (nonatomic, strong, nullable) IBInspectable UIColor *offTintColor;
```

Comment:
We should probably make a choice for consistency across the Framework, between space before and after or just before 🤔 It is gonna be a tough one though!